src/accountsdb/fuzz.zig

```diff
@@ -140,6 +158,23 @@ pub fn run(seed: u64, args: *std.process.ArgIterator) !void {
     var pubkeys_this_slot = std.AutoHashMap(Pubkey, void).init(allocator);
     defer pubkeys_this_slot.deinit();
 
+    {
+        for (&threads) |*thread| {
+            defer spawned_threads += 1;
```

Comment:
Should happen after the spawn; Erroring out would still increment it & join an uninitialized thread in the above defer